.github/workflows/end_to_end.yml

```diff
@@ -42,23 +42,23 @@ jobs:
           arguments: shadowJar
 
       #- name: Validate dataset from -- ACRONYM #<-- uncomment this line, replace ACRONYM by the name of the agency/publisher acronym
-      #  run: java -jar application/cli-app/build/libs/*.jar -u DATASET_PUBLIC_URL -i [[[ACRONYM]]].zip -e input -o output #<-- uncomment this line,
+      #  run: java -jar application/cli-app/build/libs/*.jar -url DATASET_PUBLIC_URL -input [[[ACRONYM]]].zip -extract pathToExtractedZip -output validationResultDirectory #<-- uncomment this line,
```

Comment:
Every argument is missing a dash (-). Ex. `-url` should be `--url`.

Also, does `--extract` refer to the zip file to be extracted? If yes, I would suggest modifying `pathToExtractedZip` to `pathToZipFile` or `pathToZipToExtract`.

Reply:
Actually `--extract` is the path where the content of the zip file is extracted to. It could be `pathToExtractedZipContent`...